### PR TITLE
Blogpost ready for review

### PR DESCRIPTION
**Pull request by mbogaczynska**
Hi!
Here's my blog post about history of programming languages.

Ready for review, thanks!

### CODE REVIEW

File: _posts/2020-11-07-brief-history-of-programming-languages.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
+---
+layout: post
+title:  "Brief history of programming languages"
+date:   2020-11-7 01:54:04 -0700
+categories: Programming languages
+---
+# Brief history of programming languages
+
+Early on, programming languages were highly specialized in their tasks, operating in binary codes for the exact computer. In the 20th century, computer programming became essential in the world, running the systems in almost every device we use. Humans and machines "think" differently and computer programming allowed us to tell machines what to do. Modern programming languages use syntax that is more accessible for communicating instructions.
```
Comment (LexlyCruz) [inline _posts/2020-11-07-brief-history-of-programming-languages.md:9]: Nice introduction!

File: _posts/2020-11-07-brief-history-of-programming-languages.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+![Trial model of the Analytical Engine displayed in the Science Museum in London](https://upload.wikimedia.org/wikipedia/commons/a/ac/AnalyticalMachine_Babbage_London.jpg)Trial model of the Analytical Engine displayed in the Science Museum in London
+
+## Assembly languages
```
Comment (LexlyCruz) [inline _posts/2020-11-07-brief-history-of-programming-languages.md:17]: I believe you have the same information more than once in the Assembly languages section.

File: _posts/2020-11-07-brief-history-of-programming-languages.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+
+![](https://upload.wikimedia.org/wikipedia/commons/2/22/EDSAC_%2819%29.jpg)Electronic delay storage automatic calculator (EDSAC)
+
+## Autocode
+
+Alick Glennie develped Autocode at the University of Manchester in the early 1950s, which is believed to be the first compiled programming language, which means that it can be translated directly into machine code using complier - a program primarily used for translation of high-level language to a lower level in order to create an executable program.
+
+## Fortran
+
+In 1954 Fortran (which stands for Formula Translation) was created by John Backus. It was the first widely used general purpose programming language that was used for complication mathematical, scientific and statistical work. When introduced, it was met with skepticism because of delays in development and bugs but eventually became known for its efficiency. It is one of the oldest languages still used today for programs that benchmark the world's fastest computers.
```
Comment (LexlyCruz) [inline _posts/2020-11-07-brief-history-of-programming-languages.md]: Grammar: "general-purpose"

File: _posts/2020-11-07-brief-history-of-programming-languages.md [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+
+## AGOL and COBOL
+
+Created in 1958 and 1959 respecively. Agol stands for Algorithmic Language and was created for scientific use. It is a predecessor of Pascal, C, C++, and Java. COBOL stands for COmmon Business Oriented Language and was created by  Dr. Grace Murray Hopper. It was designed for business and it is still used in ATMs, credit card processing, telephone systems, hospital and government computers, automotive systems, and traffic signals. *Fun fact*: Pieces of COBOL source code were used in Terminator's vision display in the movie "The Terminator".
```
Comment (LexlyCruz) [inline _posts/2020-11-07-brief-history-of-programming-languages.md]: Grammar: typo - "C0mmon Business Oriented Language"; extra space after "by" in "was created by Dr. Grace Murray Hopper."

File: _posts/2020-11-07-brief-history-of-programming-languages.md [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+
+Objective-C was developed in 1983 by Brad Cox and Tom Love and it is a general-purpose programming language that adds Smalltalk-style messaging to the language. It is the main language supported by Apple for macOS and iOs, the operating system of Apple.
+
+## C++
+
+Developed by Bjarne Stroustrup in 1983 and based on C language, C++ is one of the most widely used languages in the world. Game engines and high-performance software such as Adobe Photoshop are in C++.
+
+## Perl
+
+Perl was developed by Larry Wall in 1987 as a scripting language to make report processing easier. Nowadays it is used for Linux system administration, Web development  and network programming. Perl is actually a family of languages that include features from other languages including C, shell script, AWK and sed. Perl 5 became popular in late 1990s as a CGI scripting language, due to its regular expression and string parsing abilities.
```
Comment (LexlyCruz) [inline _posts/2020-11-07-brief-history-of-programming-languages.md]: Extra space after "development" in "Web development  and network programming"

File: _posts/2020-11-07-brief-history-of-programming-languages.md [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+
+## Python
+
+Guido Van Rossum designed Python in 1991. The premise of Python was that is is easier to read for humans and it requires fewer lines of code in comparison to other programming languages. Nowadays Python is used for sites such as Instagram, Uber, Spotify or Dropbox. *Fun fact 2:* Python was named after the Monty Python, the British comedy group.
```
Comment (LexlyCruz) [inline _posts/2020-11-07-brief-history-of-programming-languages.md]: Grammar: typo - "The premise of Python was that is is easier to read"

**Review by LexlyCruz — CHANGES_REQUESTED** [review]
Great job overall! I just had a few suggestions for improvements.

File: _posts/2020-11-07-brief-history-of-programming-languages.md (hunk 4 above)
Comment (mbogaczynska) [reply _posts/2020-11-07-brief-history-of-programming-languages.md]: CO is intentionally capitalized to show where the abbreviation COBOL came from. Thanks for the double space remark!

File: _posts/2020-11-07-brief-history-of-programming-languages.md (hunk 2 above)
Comment (mbogaczynska) [reply _posts/2020-11-07-brief-history-of-programming-languages.md:17]: Thanks for the remark! (Version control is important!)

File: _posts/2020-11-07-brief-history-of-programming-languages.md (hunk 3 above)
Comment (mbogaczynska) [reply _posts/2020-11-07-brief-history-of-programming-languages.md]: Thanks!

File: _posts/2020-11-07-brief-history-of-programming-languages.md (hunk 5 above)
Comment (mbogaczynska) [reply _posts/2020-11-07-brief-history-of-programming-languages.md]: Thanks!

File: _posts/2020-11-07-brief-history-of-programming-languages.md (hunk 6 above)
Comment (mbogaczynska) [reply _posts/2020-11-07-brief-history-of-programming-languages.md]: Thanks! Fixed!

**Comment by melt6457** [comment]
@mbogaczynska awesome blog post!! I only have one comment. Can you remove .DS_Store from the main project directory? This could cause merge conflicts if multiple people push that file there. Once you do this message me here or on slack and I will merge your post!

@LexlyCruz nice job on this formal review!!

**Review by melt6457 — APPROVED** [review]
@mbogaczynska I just wanted to add my formal approval to this pull request!!

**Pull request merged**